src/ControlsManager.ts

```diff
@@ -0,0 +1,59 @@
+export enum Control {
+  up = 'up',
+  down = 'down',
+  left = 'left',
+  right = 'right',
+  use = 'use'
+}
+
+const configuration: Record<string, Control> = {
+  KeyD: Control.right,
+  KeyA: Control.left,
+  KeyW: Control.up,
+  KeyS: Control.down,
+  KeyE: Control.use
+};
```

Comment:
Better to use satisfies here, it'll preserve the type info better. I.e. TS will continue to know about the keys in controls in a type safe manner.